test/controllers/YourBillControllerSpec.scala

```diff
@@ -157,20 +158,22 @@ class YourBillControllerSpec extends ItSpec {
       tableRows(1).select(".govuk-summary-list__value").text() shouldBe "£1,000"
     }
 
-    "return your bill page for SA" in {
+    "return your bill page for SA for known MainTrans code" in {
```

Comment:
it'd be nice to have a test for each main trans code if we can just so we've got the coverage on the conversion from code to content on the page somewhere